packages/twenty-front/src/modules/object-record/object-options-dropdown/states/objectOptionsDropdownRecordGroupHideComponentState.ts

```diff
@@ -0,0 +1,9 @@
+import { createComponentStateV2 } from '@/ui/utilities/state/component-state/utils/createComponentStateV2';
+import { ViewComponentInstanceContext } from '@/views/states/contexts/ViewComponentInstanceContext';
+
+export const objectOptionsDropdownRecordGroupHideComponentState =
+  createComponentStateV2<boolean>({
```

Comment:
I wonder if all these states should be record-index level and the options should write into it
recordIndexRecordGroupHide. It feels that the optionsMenu is just the UI but the information belongs to the record-index